db/migrate/20150907131503_create_task_groups.rb

```diff
@@ -0,0 +1,20 @@
+class CreateTaskGroups < ActiveRecord::Migration
+  def up
+    create_table :foreman_tasks_task_groups do |t|
+      t.string :type, index: true, null: false
+      t.integer :recurring_logic_id, index: true
```

Comment:
Shouldn't this association param be on the `recurring_logics` to avoid the sparse table (other TaskGroup STI tables would not care about this